src/ontology/cl-odk.yaml

```diff
@@ -161,4 +163,6 @@ components:
         - cellxgene_subset.tsv
     - filename: clm-cl.owl
       source: https://raw.githubusercontent.com/Cellular-Semantics/CellMark/main/clm-cl.owl
+    - filename: wmbo-cl-comp.owl
+      source: https://raw.githubusercontent.com/Cellular-Semantics/whole_mouse_brain_ontology/cl_product/wmbo-cl-comp.owl
```

Comment:
Likewise: why not downloading from `https://purl.brain-bican.org/ontology/wmbo/wmbo-cl-comp.owl`?